Various docs improvements from JOSS review (#126)
OceanBioME/OceanBioME.jl: README.md

Reviewer: We also need to update the video link to the one made with the updated resolution.

Author: We could probably update it to show the one generated in the docs now instead of being a manual upload

Author: I had a think about this and don't think there's a way for us to get it from the correct branch/preview docs. Do you think we should change it to always get the video from the stable docs? This would only be an issue if we change the video and want to preview it

Reviewer: Yes! From the stable is better. Or the dev version of docs

```

## Running your first model

> As a simple example lets run a Nutrient-Phytoplankton-Zooplankton-Detritus (NPZD) model in a two-dimensional simulation of a buoyancy front. This also requires Oceananigans, so we will add that first:

Reviewer: ```suggestion
As a simple example lets run a Nutrient-Phytoplankton-Zooplankton-Detritus (NPZD) model in a two-dimensional simulation of a buoyancy front. This example requires Oceananigans, so we install that first:
```
